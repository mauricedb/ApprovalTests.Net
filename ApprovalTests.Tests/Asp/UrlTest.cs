using ApprovalTests.Asp;
using NUnit.Framework;
using ApprovalUtilities.Utilities;

namespace ApprovalTests.Tests.Asp
{
	[TestFixture]
	public class UrlTest
	{
		[Test]
		public void TestUrlResolving()
		{
			PortFactory.AspPort = 1000;
			var urls = new[]{"~/home","/home","http://www.google.com"};
			Approvals.VerifyAll("Resolving:", urls,
				u => "{0} => {1}".FormatWith(u,AspApprovals.ResolveUrl(u)));
		}
	}
}