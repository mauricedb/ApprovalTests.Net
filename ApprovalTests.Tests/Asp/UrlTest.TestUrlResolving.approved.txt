Resolving:

~/home => http://localhost:1000/home
/home => http://localhost:1000/home
http://www.google.com => http://www.google.com
